### PR TITLE
MRG, MAINT: Add tests

### PR DESCRIPTION
**Pull request by larsoner**
Follow-up to #6976 and #6981 to add ~~a test~~ two tests. Turns out when things are degenerate there are some other minor things that need to be fixed/skipped.

### CODE REVIEW

**Comment by agramfort** [comment]
thx @larsoner 

Sorry if I merged too quickly

**Comment by larsoner** [comment]
Just pushed a commit for #6981 as well :)

File: mne/io/nirx/tests/test_nirx.py [hunk 1]
```diff
@@ -138,10 +139,14 @@ def test_nirx_15_2():
...
+@pytest.mark.parametrize('fname, boundary_decimal', (
+    [fname_nirx_15_2_short, 1],
+    [fname_nirx_15_2, 0]
+))
+def test_nirx_standard(fname, boundary_decimal):
     """Test standard operations."""
-    _test_raw_reader(read_raw_nirx, fname=fname_nirx_15_2_short,
-                     boundary_decimal=1)  # low fs
+    _test_raw_reader(read_raw_nirx, fname=fname,
+                     boundary_decimal=boundary_decimal)  # low fs
```
Comment (larsoner) [inline mne/io/nirx/tests/test_nirx.py]: cc @rob-luke this would have been my comment on #6981

Comment (rob-luke) [reply mne/io/nirx/tests/test_nirx.py]: Thanks, yes I missed that (again).

**Pull request merged**